.travis.yml

```diff
@@ -1,9 +1,19 @@
 language: python
+
 python:
   - "3.4"
+
 addons:
   postgresql: "9.4"
+
+before_install:
+    pip install codecov
+
+after_success:
+    codecov
+
 before_script:
   - psql -c 'create database "cfdm-unit-test";' -U postgres
   - pip install -r requirements.txt
+
 script: nosetests
```

Comment:
In the codecov repo I see a `nose.cfg`... maybe it uses that somehow? https://github.com/codecov/codecov-python/blob/master/nose.cfg